### ISSUE TITLE
Font#size on High DPI screens doesn't obey overall zoom in Unity

### ISSUE THREAD

**Issue by rubo77** [projecthamster/hamster#289]
On Ubuntu on my Hi-res screen some fonts are resized and some are unreadeable small

I use scale factor x2

**Comment by chri2** [comment]
Had the same problem. Even though I don't know python, worked my way through the code and found the place where fixed fonts are used for totals:

`diff /usr/lib/python2.7/dist-packages/hamster/lib/charting.py.orig /usr/lib/python2.7/dist-packages/hamster/lib/charting.py`
```
37,38c37,38
<         self.label = graphics.Label(value, size=8, color=label_color)
<         self.label_background = graphics.Rectangle(self.label.width + 4, self.label.height + 4, 4, visible=False)
---
>         self.label = graphics.Label(value, size=14, color=label_color)
>         self.label_background = graphics.Rectangle(self.label.width + 7, self.label.height + 7, 7, visible=False)
56c56,57
<         horiz_offset = min(10, self.label.y * 2)
---
>         # horiz_offset = min(10, self.label.y * 2)
>         horiz_offset = 5 
141c142
<             label = graphics.Label(stuff.escape_pango(key), size = 8, alignment = pango.ALIGN_RIGHT)
---
>             label = graphics.Label(stuff.escape_pango(key), size = 14, alignment = pango.ALIGN_RIGHT)
167c168
<         self.plot_area.x = legend_width + 8
---
>         self.plot_area.x = legend_width + 14 
241c242
<         self.graph_x += 8 #add another 8 pixes of padding
---
>         self.graph_x += 14 #add another 8 pixes of padding
248c249
<         default_font.set_size(8 * pango.SCALE)
---
>         default_font.set_size(14 * pango.SCALE)```

In my setup the font in this place at least is readable.

**Comment by elbenfreund** [comment]
Thank you for your feedback, it is much appreciated.

Project hamster and its various sub-components is currently undergoing some major changes. We prepare the introduction of a rewritten codebase for most of the underlying functionality. A direct consequence of this is that it is unlikely that any open/new bugs within the current/old codebase will be fixed (unless someone steps up and offers to do so) as most resources currently available will be invested in making the rewrite prime time ready.

**Comment by ederag** [comment]
Thanks for the investigation.
This issue seems related to the old hamster-applet (1.0.4 ?)
This seems fixed now, probably thanks to the v2 gtk3 migration
(tested on openSUSE Leap-15, 2560x1440, [v2.1.1](https://github.com/projecthamster/hamster/releases/tag/v2.1.1))

**Comment by ederag** [comment]
Fix in a better way by @GeraldJansen in PR #426, thanks !
